packages/react-lowcode/src/codegen/list.ts

```diff
@@ -17,32 +18,81 @@ export function isSelectedDataTable(sourceCode:string, tablePosition: SourceLine
 export async function addColumn(typesSourceCode: string, 
     io: CodeRW, 
     sourceCode:SourceLineCol, 
-    options: InsertOptions): Promise<string | undefined>{
+    options: InsertOptions, 
+    code?: string): Promise<string | undefined>{
```

Comment:
I do not think we need an new parameter. Please check `sourceCode`and `io` params.